Check

assilin opened the pull request.

assilin committed:
Delete styles.css.map

assilin committed:
Delete styles.scss

elcodabra committed:
Merge branch 'main' into check

elcodabra committed:
adds files

elcodabra committed:
adds files

elcodabra commented on images/photo.png:
свою картинку поставь ))

elcodabra commented on index.html:
```diff
@@ -0,0 +1,215 @@
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>News Ananlyser</title>
```
проверь орфографические ошибки, а лучше подключи плагин для автоматической подсветки синтаксических ошибок в своей IDE

elcodabra commented on index.html:
```diff
@@ -0,0 +1,215 @@
+</head>
+<body>
+    <header class="header">
+            <div class="header__image">
```
сделай одинаковые отступы, а лучше найди способ как автоматически IDE поправляла отступы

elcodabra replied:
а вообще советую выставлять отступ (tab) в 2 пробела

elcodabra commented on index.html:
```diff
@@ -0,0 +1,215 @@
+            </div>
+    </div>
+    </header>
+    <br/>
```
лучше для отступов использовать css

assilin replied:
да, разумеется. Здесь и чуть ниже, чтобы отделить разделы с разных страниц макет - поиск, не найдено и результаты поиска.

elcodabra commented on index.html:
```diff
@@ -0,0 +1,215 @@
+        <div class="search-result__inner">
+            <div class="search-result__searching">
+                <div class="search-result__searching-image">
+                    <img src="images\Ellipse.png" alt="not found">
```
alt для картинки лучше давать поясняющий что на картинке

elcodabra replied:
проверь все altы

elcodabra commented on index.html:
```diff
@@ -0,0 +1,215 @@
+                        <a class="footer-menu__link" href="#">About the Project</a>
+                    </li>
+                    <li class="footer-menu__item">
+                        <a class="footer-menu__link" href="#">Yandex.Practicum</a>
```
здесь лучше ValenciaJS

elcodabra commented on index.html:
```diff
@@ -0,0 +1,215 @@
+
+    <footer class="footer">
+        <div class="footer__inner">
+            <a class="footer__link" href="#">© 2019  Supersite</a>
```
поменяй год на текущий и добавь свое имя

elcodabra commented on index.html:
```diff
@@ -0,0 +1,215 @@
+                    <p>About developer</p>
+                </div>
+                <div class="about-developer__description">
+                    <p>This is a block describing the developer of the project. Here you should specify your name, what you do, what development technologies you own.</p>
+                    <br/>
+                    <p>You can also talk about the training process in the Practicum, what you have learned here and what you can help potential customers.</p>
```
вот здесь добавь небольшое описание о тебе как о разработчике

elcodabra reviewed (changes requested):
пока поревьюил только в общем
html и css вернусь еще ревьюить

assilin committed:
comment corrected index.html

M-skyi commented on index.html:
```diff
@@ -0,0 +1,223 @@
+            </div>
+          </div>
+          <div class="newscard search-result__card">
+            <img class="newscard__image" src="images\image_02.png" alt="news_image_meduza">
```
alt лучше заменить просто на news image . Картинки потом будут приходить по API и будут иметь разные источники .

M-skyi commented on index.html:
```diff
@@ -0,0 +1,223 @@
+    </div>
+    </div>
+  </header>
+  <br />
```
br здесь не нужен, а эти блоки  progress и not found news можно пока что сделать display: none; чтобы не отвлекали ведь они должны появляться при условии .

M-skyi commented on static/styles.css:
```diff
@@ -0,0 +1,436 @@
+  outline: none;
+}
+
+.header {
```
У всех корневых div нужно убрать max-width: 1440px;  они должны растягиваться на всю ширину. Лучше добавить еще какой-то container или wrapper и ему дать max-width: 1232px; там насколько я помню было 1232px лучше проверь

M-skyi commented on static/styles.css:
```diff
@@ -0,0 +1,436 @@
+
+.header {
+  max-width: 1440px;
+  height: 576px;
```
И  height: 576px; тоже нужно убрать по всем корневым блокам . Просто делай отступы используй margin или  padding . И тогда ты выйдешь на нужную тебе высоту

M-skyi commented on static/styles.css:
```diff
@@ -0,0 +1,436 @@
+  color: #B6BCBF;
+}
+.search-result__searching {
+  max-width: 1440px;
```
и тут по аналогии
Короче, на всех корневых блоках  height и max-width убрать

M-skyi commented on static/styles.css:
```diff
@@ -0,0 +1,436 @@
+}
+.search-result__searching {
+  max-width: 1440px;
+  height: 282px;
```
и тут

M-skyi commented on static/styles.css:
```diff
@@ -0,0 +1,436 @@
+}
+.search-result__title {
+  width: 712px;
+  position: relative;
```
тут position: relative; не нужен используй  Flexbox и margin

M-skyi commented on static/styles.css:
```diff
@@ -0,0 +1,436 @@
+  line-height: 46px;
+  color: #1A1B22;
+}
+.search-result__look-analyst {
```
и тут тоже position: relative; не нужен

M-skyi commented on images/image_01.png:
картинки лучше переименовать в news_image

M-skyi commented on images/georgia-de-lotz.png:
тут например bg_image

M-skyi reviewed (commented):
В общем, нужно сначала поправить сам layout убрав max-width и height . Потом позиционировать контент используя FlexBox  https://tproger.ru/translations/how-css-flexbox-works/

assilin committed:
Откорректировал по замечаниям:
- убрал br, добавил display:none
- убрал ширину и высоту в корневых блоках, отступы сделал с помощью margin и padding

- переименовал картинки
- поменял alt в img

assilin committed:
откорректировал wrapper и корневые блоки

assilin committed:
Откорректировал header

assilin committed:
-исправил по замечаниям (flexBox)
-убрал лишние назначения шрифтов, которые наследовались

assilin committed:
набросал вторую страницу, about, с учетом
замечаний по первой, только без карусели.
Я с bootstrap как то раз делал, но это то, я так понимаю.

M-skyi commented on about.html:
```diff
@@ -0,0 +1,230 @@
+        <nav class="menu header__menu">
+          <ul class="menu__list">
+            <li class="menu__item">
+              <a class="link menu__link" href="#">Main</a>
```
тут надо путь добавить на главною страницу

M-skyi commented on index.html:
```diff
@@ -0,0 +1,219 @@
+
+<body>
+  <header class="header">
+    <div class="header__image">
```
тут лишний див header__image, картинку можно повесить на див header . И у меня картинка bg_image.png не отображается проверь путь если у тебя тоже не отображается .

M-skyi commented on index.html:
```diff
@@ -0,0 +1,219 @@
+                <a class="link menu__link_active-page" href="#">Main</a>
+              </li>
+              <li class="menu__item">
+                <a class="link menu__link" href="#">About the Project</a>
```
тут добавить путь на другую страницу

M-skyi commented on index.html:
```diff
@@ -0,0 +1,219 @@
+          <div class="search-form header__search-form">
+            <div class="search-form__search-field">
+              <div class="search-form__search-placeholder">
+                <p>Enter news subject</p>
```
тут должен должен быть input

M-skyi commented on index.html:
```diff
@@ -0,0 +1,219 @@
+              </div>
+            </div>
+            <div class="search-form__search-button">
+              <p class="search-form__search-button-name">Search</p>
```
тут должен быть тег button

M-skyi commented on index.html:
```diff
@@ -0,0 +1,219 @@
+        <div class="search-result__bottom">
+          <div class="search-result__more-button">
+            <div class="search-result__more-button-name">
+              <p>More results</p>
```
и тут тег button

assilin committed:
откорректировал тэги button и input,
ссылку на bg_image
добавил карусель

assilin committed:
Добавить paper.html, paper.css, мелкие правки по другим страницам

assilin committed:
Откорректировать подписи к графику

assilin committed:
Fix links

assilin committed:
fix last reviews
